### PR TITLE
should make all tests available

### PR DESCRIPTION
**Pull request by nuin**
Changing the name of the files should make pytest run without invoking any parameters

### CODE REVIEW

**Comment by nuin** [comment]
@adamjohnwright Do not merge yet, I will add more things to the tests.

**Comment by nuin** [comment]
Take a look, it should be good to merge @adamjohnwright. I have some other code that I am adding today, don't need to delete the branch.

**Comment by adamjohnwright** [comment]
@nuin as we have discussed we need to be able to have these validation tests be called in between generating the files and uploading them. If these PyTests can't be called in between these two steps when running the App we should transfer the file validation logic into the main App and use PyTest for Unit tests that test the code .

File: tests/test_vcf.py [hunk 1]
```diff
@@ -110,35 +54,33 @@ def test_file_generation():
...
-    vcf_data = vcf_data_by_filename_and_id()
-    print(len(vcf_data))
-    for path in vcf_data:
+    print(len(vcf_data_by_filename_and_id))
+    for path in vcf_data_by_filename_and_id:
         filename = os.path.basename(path)
         print(filename)
         examples = EXAMPLE_CASES.get(filename)
         if not examples:
             print('No examples for ', filename, ', skipping ...')
```
Comment (adamjohnwright) [inline tests/test_vcf.py:81]: should use logger here too.

**Pull request closed without merging**